example/example.ts

```diff
@@ -38,18 +43,103 @@ const changeExample = (selectedExample: string) => {
   }
 
   body.appendChild(jsonForms);
+
+};
+
+const changeStyle = (style) => {
+  $('select').material_select('destroy');
+  if (style === 'bootstrap') {
+    bootstrap();
+  } else if (style === 'materialize') {
+    material();
+  } else {
+    none();
+  }
 };
+
 window.onload = (ev) => {
+
+  // create select element for selecting the example to be displayed
   const examplesDiv = document.getElementById(exampleDivId);
```

Comment:
maybe we should split this part in multiple classes so that it is easier to extend,
so in the end the onload would delegate to someone to create a dropdown for all available examples and allow to switch those, also delegate to someone who can do the same for styles and to someone for themes.